routes/config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  consumer_key: '...'
+  , consumer_secret: '...'
+  , access_token: '...'
+  , access_token_secret: '...'
```

Comment:
I would write all of these variables in a `.env` file and have that be set using the node module `dotenv`. That way it is the developer who is launching an local instance of your project to ensure that set up their development environment correctly ( with the instruction specified in your README.md)